documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -482,30 +482,46 @@ a|Records the completed snapshot in the connector offsets.
 
 |===
 
-include::{partialsdir}/modules/all-connectors/ref-connector-incremental-snapshot.adoc[leveloffset=+3]
+// Type: concept
+// ModuleID: debezium-mysql-ad-hoc-snapshots
+[id="mysql-ad-hoc-snapshots"]
+==== Ad hoc snapshots
+include::{partialsdir}/modules/all-connectors/con-connector-ad-hoc-snapshots.adoc[leveloffset=+3]
+
+// Type: concept
+// ModuleID: debezium-mysql-incremental-snapshots
+[id="mysql-incremental-snapshots"]
+==== Incremental snapshots
+include::{partialsdir}/modules/all-connectors/con-connector-incremental-snapshot.adoc[leveloffset=+3]
 
-==== Read-only Incremental snapshot
+// Type: concept
+// ModuleID: debezium-mysql-read-only-incremental-snapshots
+[id="mysql-read-only-incremental-snapshots"]
+==== Read-only incremental snapshots
```

Comment:
I think we should keep this community only. @gunnarmorling Your opinion?

Reply:
I know that we aren't making read-only incremental snapshots available downstream in Q1, but I assumed that part would be filtered out in the backport. Probably easier to conditionalize it, if we don't expect to support it for a longer period. Let me know how to proceed.